app/assets/javascripts/exchanges/carriers.js

```diff
@@ -45,11 +45,11 @@ $(document).on('turbolinks:load', function() {
                 showRow = false;
             }
 
-            if (pvpRatingAreas.length > 0 && !pvpRatingAreas.includes(pvpAreas)) {
+            if (pvpRatingAreas.length > 0 && !pvpRatingAreas.some(pra => pvpAreas.includes(pra))) {
```

Comment:
seems like a string compare, espectially of numbers, via `includes` could easily get bad matches (`['1'].some(pra => "11".includes(pra))`), unless they are perhaps all of a uniform length.

Maybe something like `pvpRatingAreas.some(pra => pvpAreas.split(/, ?/).includes(pra))` ?

I'd probably just do that for plantype and metallevel comparisons here as well, for clarity and safety.